pkg/clusterconfig/clusterconfig.go

```diff
@@ -116,30 +115,37 @@ func GetAWSConfig() (*Config, error) {
 	}
 
 	// Look for a user defined secret to get the AWS credentials from first
-	sec, err := client.Secrets(imageRegistryNamespace).Get(userPrivateConfigName, metav1.GetOptions{})
+	sec, err := client.Secrets(regopapi.OperatorNamespace).Get(regopapi.ImageRegistryPrivateConfigurationUser, metav1.GetOptions{})
 	if err != nil && errors.IsNotFound(err) {
-		glog.Infof("Optional user defined AWS credentials in secret \"%s/%s\" not found, ignoring.", imageRegistryNamespace, userPrivateConfigName)
+		glog.Infof("Optional user defined AWS credentials in secret \"%s/%s\" not found, ignoring.", regopapi.OperatorNamespace, regopapi.ImageRegistryPrivateConfiguration)
 		// If no user defined secret is found, use the system one
 		sec, err = client.Secrets(installerConfigNamespace).Get(installerAWSCredsName, metav1.GetOptions{})
 		if err != nil {
-			return nil, fmt.Errorf("unable to get secret  \"%s/%s\": %v", installerConfigNamespace, installerAWSCredsName, err)
+			return nil, fmt.Errorf("unable to get secret %q: %v", fmt.Sprintf("%s/%s", installerConfigNamespace, installerAWSCredsName), err)
 		}
-		glog.Infof("Using cluster defined AWS credentials from secret \"%s/%s\"", installerConfigNamespace, installerAWSCredsName)
 		if v, ok := sec.Data["aws_access_key_id"]; ok {
 			cfg.Storage.S3.AccessKey = string(v)
+		} else {
+			return nil, fmt.Errorf("Secret %q does not contain required key \"aws_access_key_id\"", fmt.Sprintf("%s/%s", regopapi.OperatorNamespace, regopapi.ImageRegistryPrivateConfiguration))
```

Comment:
sec is a secret from the installer.

Reply:
?

Reply:
The error mentions `fmt.Sprintf("%s/%s", regopapi.OperatorNamespace, regopapi.ImageRegistryPrivateConfiguration)`, but the lookup was in the installer secret.

Reply:
ah yes, good catch!

Reply:
updated in next push